src/test/it/initScripts/99-saveToDisk.groovy

```diff
@@ -0,0 +1,3 @@
+import jenkins.model.Jenkins
+
+Jenkins.instance.save()
```

Comment:
Why is this necessary? Every setter to a `Saveable` is expected to save the object when modified.

Reply:
C&P from CWP demo scripts